packages/realm-server/tests/helpers/index.ts

```diff
@@ -142,30 +144,33 @@ export async function createRealm({
   }
 
   let adapter = new NodeAdapter(dir);
-  let realm = new Realm({
-    url: realmURL,
-    adapter,
-    getIndexHTML: async () =>
-      readFileSync(join(distPath, 'index.html')).toString(),
-    matrix: matrixConfig,
-    permissions,
-    realmSecretSeed: "shhh! it's a secret",
-    virtualNetwork,
-    dbAdapter,
-    queue,
-    onIndexer: async (indexer) => {
-      let worker = new Worker({
-        realmURL: new URL(realmURL!),
-        indexer,
-        queue,
-        realmAdapter: adapter,
-        runnerOptsManager: manager,
-        loader: realm.loaderTemplate,
-        indexRunner,
-      });
-      await worker.run();
+  let realm = new Realm(
+    {
+      url: realmURL,
+      adapter,
+      getIndexHTML: async () =>
+        readFileSync(join(distPath, 'index.html')).toString(),
+      matrix: matrixConfig,
+      permissions,
+      realmSecretSeed: "shhh! it's a secret",
+      virtualNetwork,
+      dbAdapter,
+      queue,
+      onIndexer: async (indexer) => {
+        let worker = new Worker({
+          realmURL: new URL(realmURL!),
+          indexer,
+          queue,
+          realmAdapter: adapter,
+          runnerOptsManager: manager,
+          loader: realm.loaderTemplate,
+          indexRunner,
+        });
+        await worker.run();
+      },
     },
-  });
+    { deferStartUp },
```

Comment:
indenting made this diff hard to read, this line was the only change here

Reply:
<img width="313" alt="boxel 2024-05-15 20-57-06" src="https://github.com/cardstack/boxel/assets/43280/56597d77-541f-41ff-8d46-2f005bedcc5b">

<img width="1061" alt="boxel 2024-05-15 20-57-59" src="https://github.com/cardstack/boxel/assets/43280/7e94b13c-dc55-4cbd-8293-30d2f46e2d05">

“Hide whitespace” to the rescue!